modules/core/src/main/java/org/apache/ignite/internal/client/thin/TcpClientCache.java

```diff
@@ -127,6 +127,10 @@ public class TcpClientCache<K, V> implements ClientCache<K, V> {
     public static final String NON_TRANSACTIONAL_CLIENT_CACHE_CLEAR_IN_TX_ERROR_MESSAGE = "Failed to invoke a " +
         "non-transactional ClientCache clear operation within a transaction.";
 
+    /** Exception thrown when a non-transactional ClientCache clear operation is invoked within a transaction. */
+    public static final String NON_TRANSACTIONAL_CLIENT_CACHE_REMOVEALL_IN_TX_ERROR_MESSAGE = "Failed to invoke a " +
```

Comment:
Duplication of message. Let's use parameterized error message with an operation as a param.